src/broker/overlay.c

```diff
@@ -1793,6 +1829,25 @@ static int overlay_configure_torpid (struct overlay *ov)
                          set_torpid,
                          ov) < 0)
         return -1;
+
+    /* Only allow command line override, not runtime on these attrs
+     */
+    if (overlay_configure_attr_int (ov->attrs,
+                                    "tbon.keepalive_cnt",
```

Comment:
Same comment about the broker attributes - I don't see the value in abbreviating `count` to `cnt`.

Also it seems like it would be really handy to be able to tune these values at runtime without requiring a broker restart, but I'm guessing this is difficult because of the way the `zsock` options have to be set?

Reply:
Unfortunately these have to be set before the socket is bound, so there's no way to change it at runtime.